sysl2/sysl/fmt_grammar/FmtParser.interp

```diff
@@ -0,0 +1,48 @@
+token literal names:
```

Comment:
Add generated files to a `.gitattributes` file. E.g.: https://github.com/anz-bank/sysl/blob/master/sysl2/sysl/grammar/.gitattributes

Reply:
Added.